### ISSUE TITLE
Inconsistent link to release on email from semantic-release bot 

### ISSUE THREAD

**Issue by kpturner** [semantic-release/github#812]
When a release is created, semantic release bot sends an email that looks like this

![image](https://github.com/semantic-release/github/assets/308646/f07ad6af-942d-4203-8303-c1d81295113d)

where it says `The release is available` on followed by a link to the GitHub release.

However, for one of our packages, although a release is created in GitHub, the email looks like this

![image](https://github.com/semantic-release/github/assets/308646/277e04af-f87c-48f8-b892-13cf3c5dcfc9)

So instead of a link it is effectively a piece of text. I am trying to fathom out what on earth would cause this difference. Can anybody shed any light?

**Comment by travi** [comment]
Is a release actually created under releases/ for that repo. Seems most likely that it is failing to create the GitHub release, so there would be nothing to link to

**Comment by kpturner** [comment]
Yeah the release is being created as expected. I can't work out what makes this happen. I guess the code that generates this string might provide some clue 🤷

**Comment by travi** [comment]
is this a public project that you can link to?

**Comment by kpturner** [comment]
No unfortunately not 👎
